apps/condo/domains/organization/components/CreateEmployeeForm.tsx

```diff
@@ -126,7 +126,7 @@ export const CreateEmployeeForm: React.FC = () => {
                     values.role = { id: String(role) }
                 }
                 if (specializations) {
```

Comment:
Strange place to connect ids - can you move this logic to convertToGQLInput?

Reply:
done